lib/ansible/modules/cloud/vmware/vmware_resource_pool_info.py

```diff
@@ -124,16 +125,18 @@ def gather_rp_facts(self):
                 runtime_memory_max_usage=rp.summary.runtime.memory.maxUsage,
             )
 
-            resource_pool_facts.append(tmp_facts)
+            resource_pool_facts.append(tmp_info)
```

Comment:
```suggestion
            resource_pool_info.append(tmp_info)
```
And the return below